shenyu-examples/shenyu-examples-springcloud/pom.xml

```diff
@@ -70,11 +70,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
-            <version>${spring-cloud.version}</version>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
used spring profiles active eureka , then dependency jar

Reply:
I create application-eureka.yml,and activited nacos profiles , so  annotation this eureka pom